@xen-orchestra/lite/src/stories/web-core/user-link.story.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties, settings }"
+    :params="[
+      prop('route').type('RouteLocationRaw'),
```

Comment:
We should add a default link, to be able see the hover/pressed/disabled states.

```suggestion
      prop('route').type('RouteLocationRaw').preset('#'),
```